README.md

```diff
@@ -34,41 +34,59 @@
 - [ ]  light mode
 
 ------------
-~* README template *~ 
 # Wax Staxx
 
-Foobar is a Python library for dealing with word pluralization.
+Welcome to Wax Staxx! -  a React web application that provides information about various vinyl albums, allowing users to showcase their personal collection. for collectors it fun to talk about music and what albums you have this website is clean and to the point to be able to keep your collection up to date. 
```

Comment:
How about:
Welcome to Wax Staxx! - A React app that provides information about various albums, allowing users to track and showcase their personal collection.